__tests__/app.test.js

```diff
@@ -105,6 +107,12 @@ describe("GET /api/articles/:article_id", () => {
             expect(text).toEqual("{\"status\":400,\"error\":\"Bad Request\",\"message\":\"Invalid query\"}");
         })
     });
+    test("Should respond with 400 and send an error message if invalid id input", () => {
```

Comment:
The test descriptions where you are testing for a decimalised numbers specifically should say this explicitly